test/tools/Modules/PSSysLog/PSSysLog.psd1

```diff
@@ -0,0 +1,10 @@
+@{
+ModuleVersion = '1.0.0'
+GUID = '56b63338-045c-4697-a24b-5a756268c8b2'
+Author = 'Dan Travison'
+CompanyName = 'Microsoft'
+Copyright = 'Copyright (c) Microsoft Corporation. All rights reserved.'
```

Comment:
? Are you saying there should not be a copyright here?  Every other module under test\tools\modules has copyright information.

Reply:
I haven't full understanding that is a policy because the product is under MIT but many files contains Microsoft copyright. CoreFX is under MIT and has MIT in files. Anyway all the new files we created last months do not contain Microsoft copyright and MIT copyright too. /cc @SteveL-MSFT 
I would prefer to see in our contribution documents a direct policy statement.

Reply:
@iSazonov can you open a new issue for Copyrights?  I agree that we've been inconsistent and I can follow up with legal and have a specific statement in our docs

Reply:
Done #6073.